luda-editor/luda-editor-client/src/app/app.rs

```diff
@@ -0,0 +1,122 @@
+use wasm_bindgen_futures::spawn_local;
+
```

Comment:
@bigfoodK There are some cases that put additional line break between `use`. Please remove that line breaks like this.

Reply:
Done with ffb49c2